src/assets/locales/en.json

```
@@ -2671,14 +2671,17 @@
         }
       },
       "daoSettingsInfo": {
+        "address": "Address",
+        "blockchain": "Blockchain",
         "chain": "Chain",
-        "ens": "ENS name",
-        "links": "Resources",
+        "ens": "ENS Name",
```

I don't agree but we actually have a pattern of not capitalizing all words in a title/label/button etc
```suggestion
        "ens": "ENS name",
```

---

I changed it because of figma https://www.figma.com/design/LFe79BV9DoC1VB96FvjT7d/APP-278--Capital-flow-primitives?node-id=17571-24779&m=dev